src/igvfd/schemas/reference_file.json

```diff
@@ -245,17 +248,6 @@
                 "igvf_utils": "vcf"
             }
         },
-        "assembly": {
```

Comment:
I think this part can be kept, and only remove the enum list in this.

Reply:
done